File: eclipse/idm.project.tests/ressources/expected/r/removeLine.R
# Creates a new DataFrame from a CSV File
df = read.csv(file = "C:/Users/Leloup/Documents/FAC/M2/IDM/Git/IDM-Projet/eclipse/idm.project.tests/ressources/input/fruits.csv")
#removes a row at the given index
df <- df[-c(5), ]

write.csv(df,"C:/Users/Leloup/Documents/FAC/M2/IDM/Git/IDM-Projet/eclipse/idm.project.tests/ressources/output/removeLineR.csv",quote = FALSE,row.names = FALSE)

File: eclipse/idm.project.tests/ressources/expected/python/removeLine.py
import pandas as pd
df = pd.read_csv( "/home/simonlecordier/workspace/m1/IDM/IDM-Projet/eclipse/idm.project.tests/ressources/input/fruits.csv" )
df = df.drop(5)
df.to_csv("/home/simonlecordier/workspace/m1/IDM/IDM-Projet/eclipse/idm.project.tests/ressources/output/removeLinePython.csv", index=False)

